### DISCUSSION TITLE
Swapping is error

### DISCUSSION THREAD

**Discussion by starman1121** [community/community#36411] [Other Feature Feedback, Questions, & Ideas] [answered]
I am getting error in swapping.
I created swap by forking uniswap v3.
who can help me?

**Comment by kryptoDev0777** [comment]
Hello, nice to meet you.
give me your website link, I will check on my side

**Reply by starman1121** [reply]
Thanks for your quick reply
https://app.buddhaswap.io

**Comment by kryptoDev0777** [comment]
yeah, let me check

**Reply by starman1121** [reply]
Thanks man

**Comment by kryptoDev0777** [comment]
hi, it works well in my side.
what did you try on swap?

**Reply by starman1121** [reply]
tried just swapping

**Comment by kryptoDev0777** [comment]
add liquidity is working?

**Reply by starman1121** [reply]
it's also not working

**Comment by kryptoDev0777** [comment]
do you have enough gas?

**Reply by starman1121** [reply]
0.002 ETH

**Comment by kryptoDev0777** [comment, marked as answer]
oh, man, it's too low, please try with good amount

**Reply by starman1121** [reply]
ah~ you are right, it works on my other wallet
Thanks man

**Comment by kryptoDev0777** [comment]
no problem, If you have any other issue, let me know I will help you surely

**Reply by starman1121** [reply]
Thanks again

**Comment by ossanna16** [comment]
Great question @starman1121, I'm glad you found our community 🚀

**Comment by ossanna16** [comment]
And thank you for helping @mgx5kw, we appreciate it!

**Reply by kryptoDev0777** [reply]
Thanks, could you follow me?

**Reply by kryptoDev0777** [reply]
and I want to get job though this community
is it possible?

**Reply by ossanna16** [reply]
You can find all open GitHub jobs here https://github.com/about/careers. Best of luck!
